### PR TITLE
New reflect_Actor test

### PR DESCRIPTION
**Pull request by tbkka**
Make sure that Actor reflection continues to work.  (This was reported as broken a little while ago, but seems to work now; I presume that means something fixed it at some point?)

Related to rdar://98686546

### CODE REVIEW

**Comment by tbkka** [comment]
@swift-ci Please test

**Comment by tbkka** [comment]
@swift-ci Please test macOS Platform

**Comment by tbkka** [comment]
@swift-ci Please test

File: validation-test/Reflection/reflect_Actor.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift -lswiftSwiftReflectionTest %s -o %t/reflect_Actor.swift
+// RUN: %target-codesign %t/reflect_Actor.swift
+
+// RUN: %target-run %target-swift-reflection-test %t/reflect_Actor.swift | %FileCheck %s --check-prefix=CHECK-%target-ptrsize
+
+// REQUIRES: reflection_test_support
+// REQUIRES: executable_test
+// UNSUPPORTED: use_os_stdlib
```
Comment (mikeash) [inline validation-test/Reflection/reflect_Actor.swift:9]: This might need `UNSUPPORTED: back_deployment_runtime` too. CI will tell.

**Pull request merged**